1주차/10814/10814_python_제로.py

```diff
@@ -0,0 +1,14 @@
+n = int(input())
+
+# 리스트
+lst = []
+for i in range(n):
+    age, name = map(str, input().split())
+    lst.append((int(age), name))
+
+# 정렬
+lst.sort(key=lambda x: x[0])
+
+# 출력
+for i in range(n):
+    print(lst[i][0], lst[i][1])
```

Comment:
인덱스 대신 바로 원소를 꺼내 쓸 수도 있을 것 같아여 
```suggestion
for user in lst:
    print(user[0], user[1])
```

Reply:
킹정합니다